src/platform/window.cpp

```
@@ -17,6 +17,17 @@ namespace moth_graphics::platform {
         return GetSurfaceContext().GetAssetContext().GetImageFactory();
     }
 
+    bool Window::OnEvent(moth_ui::Event const& event) {
```

Clang (14.0.6)

[error] method 'OnEvent' can be made static

(readability-convert-member-functions-to-static,-warnings-as-errors)